implemented figma. added few widgets.

iberdinsky-skilld opened the pull request:
## Issue

https://github.com/jsdrupal/drupal-admin-ui/issues/705

## Screenshot / UI changes

Added figma integration. As [decorator](https://github.com/hharnisc/storybook-addon-figma#or-use-the-decorator-to-put-the-same-design-on-each-story) 

[![Image from Gyazo](https://i.gyazo.com/e405250bedf818e0a5421871b5ba8fce.png)](https://gyazo.com/e405250bedf818e0a5421871b5ba8fce)

Added few components figma integration. In all other existing components stories i wasn't sure about their designs location. 

## Testing instructions

Current implementation of storybook is broken, so this pr will work 
# ONLY
with that small [fix](https://github.com/jsdrupal/drupal-admin-ui/issues/632#issuecomment-459645865)

dawehner commented:
Thank you for working on this!
Note: We'll use the storybook in `packages/components` instead, as the previous version was mostly a prototype.

@iberdinsky-skilld I'm wondering whether @ckrina could split up the figma templates into a reference per component? This would make it nicer.

> Current implementation of storybook is broken, so this pr will work

Could you add a PR for itself?

iberdinsky-skilld commented:
@dawehner please recheck.
1. I added same steps to components workspace. also added teststory to only one existing button component. 

[![Image from Gyazo](https://i.gyazo.com/d88446c16487a404dba96bdf5d6874da.png)](https://gyazo.com/d88446c16487a404dba96bdf5d6874da)

2. also it is wierd but i didn't have issues with building storybook in components workspace. so maybe 

> Could you add a PR for itself?

not required anymore?

3. I think it is ok to have all modifiers in one page of figma. Storybook should display all of them aswell:

like here: https://danieldelcore.github.io/trousers/?path=/story/basic--elements-and-modifiers
